<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextFormat</class>
 <widget class="QWidget" name="TextFormat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>39</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="edSendText"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblTermChar">
     <property name="text">
      <string>Term:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinTermChar</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="spinTermChar">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string>Char </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>13</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblDescription"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
